src/screens/NewsScreen/Search.js

```diff
@@ -96,7 +96,7 @@ const styles = StyleSheet.create({
   },
   animatedViewContainer: (animatedValue) => ({
     flexDirection: 'row',
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, replacing hardcoded color values with references to a centralized `COLORS` object is a good practice for maintaining consistency and ease of updates.
```diff
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->